src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -1608,7 +1609,7 @@ private static LazyStringValue UnwrapLowerIdIfNeeded(JsonOperationContext contex
 
             if (lowerId[lowerId.Size - ConflictedTombstoneOverhead] == SpecialChars.RecordSeparator)
             {
-                return new LazyStringValue(null, lowerId.Buffer, lowerId.Size - ConflictedTombstoneOverhead, context);
+                return context.GetLazyStringValue(lowerId.Buffer);
```

Comment:
@arekpalinski I do need to parse it for the export/backup, so I ended up doing it like this.

Reply:
Are we unwrapping here anything now? It looks like it takes the whole buffer so it will be the same as `lowerId`?